src/faldbt/project.py

```diff
@@ -38,13 +39,34 @@ def normalize_directories(base: str, dirs: List[str]) -> List[Path]:
     )
 
 
+@dataclass
+class DbtTest:
+    node: Any
+    name: str = field(init=False)
+    model: str = field(init=False)
+    column: str = field(init=False)
+    status: str = field(init=False)
+
+    def __post_init__(self):
+        node = self.node
+        self.unique_id = node.unique_id
+        if node.resource_type == NodeType.Test:
+            self.name = node.test_metadata.name
+            self.model = re.findall(r"'([^']+)'", node.test_metadata.kwargs['model'])[0]
```

Comment:
Can you add a comment explaining the "typical" content of `node.test_metadata.kwargs['model']` to understand the reasoning for a regex? Also helps if this content changes in a future dbt version